x-pack/plugins/remote_clusters/public/sections/remote_cluster_list/detail_panel/detail_panel.js

```diff
@@ -186,6 +191,32 @@ export class DetailPanelUi extends Component {
         </EuiFlyoutHeader>
 
         {content}
+
+        <EuiFlyoutFooter>
+          <EuiFlexGroup justifyContent="spaceBetween">
+            <EuiFlexItem grow={false}>
+              <EuiButton color="danger">
```

Comment:
could we consolidate the Disconnect and Edit actions under a single Manage button? this will be consistent with our pattern in Index Management and Rollup Jobs